gulpfile.js

```diff
@@ -786,6 +787,49 @@ gulp.task(
   })
 );
 
+// Builds the generic production viewer that should be compatible with most
```

Comment:
I'm wondering if there is a way to merge these two very similar targets in some way. If that's difficult, it's not a must-have, but it would be nice to reduce the size of the file. Same for the target below.

Reply:
I'll try to fix this, since the duplication is mildly annoying.